src/android/plugin/google/maps/MyPluginLayout.java

```diff
@@ -281,6 +285,50 @@ public void inValidate() {
     this.frontLayer.invalidate();
   }
 
+  public void setDragging() {
```

Comment:
Maybe this method can be called `setDraggingStarted` to indicate more clearly its intent